spec/models/user_spec.rb

```diff
@@ -1,57 +1,20 @@
 require 'spec_helper'
 
-describe User do
-  before(:each) {
-    User.destroy_all
-    roles = ["Admin","Editor", "Author", "Member", "Creative", "Moderator"]
-
-    roles.each do |role|
-      Role.create!({ name: role })
-    end
-  }
-  after(:each) { User.destroy_all }
-
+RSpec.describe User do
   it 'validates the name is unique' do
-    expect(User.new({
-                      name: 'Connor',
-                      password: 'testpassword',
-                      email: 'fake@fake.com'
-                    })).to be_valid
-    User.create!({
-                   name: 'Connor',
-                   password: 'testpassword',
-                   email: 'fake@fake.com'
-                 })
-    expect(User.new({
-                      name: 'Connor',
-                      password: 'testpassword',
-                      email: 'fake@fake.com'
-                    })).to be_invalid
+    create(:user)
+    expect(build(:user)).to be_invalid
   end
 
   it 'will default the role to [3] if none is provided' do
-    user = User.create!({
-                          name: 'Alex',
-                          password: 'testpassword',
-                          email: 'fake@fake.com'
-                        })
+    user = create(:user)
     expect(user.role_ids).to eq([3])
   end
 
   describe '#make_all_members' do
     before(:each) {
-      @user = User.create!({
-                             name: 'Alex',
-                             password: 'testpassword',
-                             email: 'fake@fake.com',
-                             role_ids: [2, 3]
-                           })
-      @admin_user = User.create!({
-                                   name: 'Connor',
-                                   password: 'testpassword',
-                                   email: 'fake2@fake.com',
-                                   role_ids: [1, 2, 3]
-                                 })
+      @user = create(:user, name: 'Alex', role_ids: [2, 3])
```

Comment:
I mean really I don't mind as long as it is consistent, but I think I will configure this one. I do prefer the single quotes.